gallery/src/components/NxList/NxListPage.tsx

```diff
@@ -13,10 +13,11 @@ const NxListPage = () =>
   <>
     <GalleryDescriptionTile>
       <NxP>
-        <NxCode>NxList</NxCode> encapsulates the styles defined in the <NxCode>nx-list</NxCode> style page into
-        simple React components. In addition, <NxCode>NxList</NxCode> also contains children components that make
-        it easy to customize lists. With the help of these components, it is quite simple to create lists that have
-        clickable buttons, links, action buttons, multiple lines of text, and even lists with empty and error states.
+        <NxCode>NxList</NxCode> encapsulates the styles defined in the{' '}
+        <NxTextLink href="#/pages/List%20(HTML)">List (HTML)</NxTextLink> style page into simple React components. In
```

Comment:
I updated some of the broken links' names. I think updating them from their `nx-` names to their official name like `List 
 (HTML)` makes the documentation more clear.

Updating broken link names is not asked for in the [original ticket](https://issues.sonatype.org/browse/RSC-1101). But I went ahead and did it since its so closely related to the ticket.

Reply:
@rpokorny
Per our discussion during WW and Slack, I'm marking this as resolved. We're keeping the updated link names.